### PR TITLE
Rebrand /aws/workspaces page

### PR DESCRIPTION
**Pull request by britneywwc**
## Done

- Refresh /aws/workspaces based on [Figma design](https://www.figma.com/design/FoWXkNxxdri9EgSsLdtTrH/24.10-AWS-bubble?node-id=34-4124&m=dev)
- Reformat using djlint

## QA

- Go to https://ubuntu-com-14162.demos.haus/aws/workspaces
- Check that page design matches Figma

## Issue / Card

Fixes [WD-12849](https://warthogs.atlassian.net/browse/WD-12849)

## Screenshots

[If relevant, please include a screenshot.]

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

[WD-12849]: https://warthogs.atlassian.net/browse/WD-12849?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: templates/aws/workspaces.html [hunk 1]
```diff
@@ -1,96 +1,156 @@
...
-            alt="",
-            height="215",
-            width="603",
-            hi_def=True,
-            loading="auto"
-          ) | safe
-        }}
+      <div class="col p-strip is-shallow">
+        <div class="p-image-container--16-9 is-highlighted">
+          {{ image (url="https://assets.ubuntu.com/v1/b914870e-aws-f7f7f7.png",
```
Comment (akbarkz) [inline templates/aws/workspaces.html:34]: please remove the space between `image` and the bracket, then it should automatically format it correctly

File: templates/aws/workspaces.html [hunk 2]
```diff
@@ -1,96 +1,156 @@
...
-        <li class="p-list__item is-ticked">Pay as you go without the need to procure high-end hardware</li>
-      </ul>
-
-      <p>Read more about <a href="https://aws.amazon.com/workspaces/">Amazon WorkSpaces</a></p>
+    <div class="row">
+      <div class="col-start-large-4 col-9">
+        <hr class="p-rule is-muted" />
+        <div class="row">          
+          <div class="col-3">
+            <h5>Ubuntu Desktop 22.04 LTS</h5>
```
Comment (akbarkz) [inline templates/aws/workspaces.html:67]: it's better to not use different headings together, so `<h3 class="p-heading--5">` would be better. Please adjust it in the below sections too.

File: templates/aws/workspaces.html (hunk 1 above)
Comment (akbarkz) [reply templates/aws/workspaces.html:34]: Also, we normally do not show the image on small screens in hero sections

**Comment by akbarkz** [comment]
Please increase the space between adjacent blocks here:
Screenshot 2024-08-09 at 16 17 47

**Comment by mattea-turic** [comment]
@britneywwc it matched the figma file but I just wanted to make a few changes since the hero looks a bit odd. I added the comments directly in the file

**Comment by britneywwc** [comment]
@akbarkz could you please clarify which spaces to increase for the blocks? I am following the design on Figma and the spacing matches.
Screenshot 2024-08-12 at 12 23 02 PM
Screenshot 2024-08-12 at 12 23 19 PM

**Comment by britneywwc** [comment]
@mattea-turic thanks for the review, I've updated the changes accordingly, could you please have another look?

**Comment by akbarkz** [comment]
> @akbarkz could you please clarify which spaces to increase for the blocks? I am following the design on Figma and the spacing matches. Screenshot 2024-08-12 at 12 23 02 PM Screenshot 2024-08-12 at 12 23 19 PM

For example, between "Ubuntu Desktop 22.04 LTS" and "Remote onboarding for developers" blocks. On big screens it looks fine, but on medium and small you can't see if they are different blocks, cause they are too close to each other:
Screenshot 2024-08-12 at 09 32 45

**Comment by britneywwc** [comment]
@mattea-turic Could you also have a look at @akbarkz's feedback of the divided list on small & medium screen views?

**Comment by mattea-turic** [comment]
@britneywwc it looks a lot better on mobile/smaller screens now – thanks!

**Review by akbarkz — APPROVED** [review]

**Pull request merged**